docs/en/sdk-manual/migrate-from-third-party-sdks.md

```diff
@@ -0,0 +1,122 @@
+# Migrate from third-party SDKs
+
+## Introduction
+
+Generally, in order not to affect your normal business analysis, when migrating from a third-party SDK to Clickstream SDK, we recommend that you integrate your existing SDK with Clickstream SDK for a period of time, and then remove it when you think that Clickstream SDK can replace the previous SDK.
```

Comment:
This article provides a best practice for you to migrate from a third-party SDK to Clickstream SDK. If you already has a SDK in your app or website and you want to replace it with Clickstream SDK, we recommended you adopt this practice which provides the following benefits

- Minimum code changes
- Quick implementation time
- Dual measurement to ensure smooth cutover

Reply:
changed to this